src/System.Runtime/tests/System/LazyTests.cs

```diff
@@ -463,27 +464,35 @@ public static void EnsureInitialized_ComplexRefTypes()
             object aLock = null;
             Assert.NotNull(LazyInitializer.EnsureInitialized(ref a, ref aInit, ref aLock));
             Assert.NotNull(a);
+            Assert.True(aInit);
+            Assert.NotNull(aLock);
```

Comment:
@stephentoub I have added asserts for true / not null in earlier test cases as well. Please let me know if this is ok...